### ISSUE TITLE
AP Biology nested lists don't indent

### ISSUE THREAD

**Issue by oscryan** [openstax/cnx-recipes#5435]
# Brief Description of Scenario
They used to (new on the left)

![Image](https://github.com/openstax/cnx-recipes/assets/26280712/ee7b4b9b-8425-4616-b7eb-b359f00124c7)

# Pattern of Occurrence 

# Acceptance Criteria
- please indent nested lists
  
# Sample Mark-Up
```
<!-- Please paste here if applicable. -->
```

# Location(s)
- **osbooks repo:** osbooks-biology-bundle
- **collection slug:** biology-ap-courses
- **ref/branch:**
- **chapter/section:** 
- **surrounding text:** Pathways of Photosynthesis and Cellular Metabolism

**Comment by oscryan** [comment]
Sometimes the lack of indentation creates crashes
<img width="1169" alt="Screenshot 2024-02-07 at 4 42 57 PM" src="https://github.com/openstax/cnx-recipes/assets/1895432/2d99e628-b54b-470e-a490-186108c0bb67">

**Comment by lwtchu** [comment]
@kswanson33 pushing to next sprint. going to wait to see if the fix tyler did will fix this

**Comment by lwtchu** [comment]
![Image](https://github.com/openstax/cnx-recipes/assets/11299125/040a9c3e-6f93-4bfd-9141-382bab1bef94)

Issue still persist in newly generated PDF

**Comment by kswanson33** [comment]
From investigation: the problems shown in the above screenshots are two different problems.

1. ![Image](https://github.com/openstax/cnx-recipes/assets/26280712/842f79c9-796a-48ea-8f5e-f727964c8ab2)

There are some inconsistencies in the markup here (a `div` between nested lists; i.e. `ol > li > div > ol > li` instead of `ol > li > ol > li`). I have fixed the issue in [this](https://github.com/openstax/ce-styles/pull/514) PR.

2. ![Image](https://github.com/openstax/cnx-recipes/assets/26280712/f1a11e5e-1dfd-41d3-b7f4-256b97e7c063)

These are not nested lists, but two adjacent lists. I believe the way they are being displayed is as expected. Let me know if a change is needed.

**Comment by openstaxalina** [comment]
Understood @kswanson33 . I looked at the markup (the question is in Exercises) and the question has a list, followed by multiple choice options:

![Screenshot 2024-06-04 at 10 58 35 AM](https://github.com/openstax/cnx-recipes/assets/7863733/b15b42f5-20b4-4e66-8070-8cfb8b36120a)

The new display is confusing, and I think it will occur multiple times -- I found another instance just looking at 1 chapter of AP Bio:

## Old PDF
![Screenshot 2024-06-04 at 11 02 12 AM](https://github.com/openstax/cnx-recipes/assets/7863733/233932d7-881e-406d-9c40-c8f1135d4c5e)

## Latest PDF
![Screenshot 2024-06-04 at 11 06 28 AM](https://github.com/openstax/cnx-recipes/assets/7863733/73306684-2b0f-4fe0-926d-c892fe56477a)

I don't see a card for the period and space before the question...so **maybe we should split Kendra's second issue into a new card for a general investigation into exercise styling in AP Bio?**

**Comment by openstaxalina** [comment]
(By the way, the question in Ryan's first comment can be found by tag `apbio-ch03-ex014` in the Exercises database)

**Comment by kswanson33** [comment]
@openstaxalina That sounds good. I'd even support two cards -- one for the "question ending in a list followed by a list of answer options" issue (may need new specs) and another for the number spacing issue (should be straightforward).

I'll update the original description of this card and use it for the nested lists issue.

**Comment by openstaxalina** [comment]
created 2 spinoff cards:
- https://github.com/openstax/cnx-recipes/issues/5562
- https://github.com/openstax/cnx-recipes/issues/5563

They are both in the backlog since we don't urgently need to fix AP Bio PDF display issues (just needs to be done before ~March 2025).

**Comment by kswanson33** [comment]
QA: fix is on ce-styles `bio-nested-list`

**Comment by MartaSztuka** [comment]
@kswanson33 there still problem with indentation
![obraz](https://github.com/openstax/cnx-recipes/assets/47241114/f9f91d86-2cf8-4045-96e6-080e89c85fb6)

**Comment by MartaSztuka** [comment]
#### PASSED
![obraz](https://github.com/openstax/cnx-recipes/assets/47241114/5bacf0a6-7bf2-478c-8c45-574237eedb1c)
